tests/scripts/thread-cert/Cert_5_3_07_DuplicateAddress.py

```diff
@@ -101,26 +111,46 @@ def test(self):
         time.sleep(5)
         self.assertEqual(self.nodes[ROUTER2].get_state(), 'router')
 
-        self.nodes[ED1].start()
+        self.nodes[MED1].start()
         time.sleep(5)
-        self.assertEqual(self.nodes[ED1].get_state(), 'child')
+        self.assertEqual(self.nodes[MED1].get_state(), 'child')
 
-        self.nodes[ED2].start()
+        self.nodes[SED1].start()
         time.sleep(5)
-        self.assertEqual(self.nodes[ED2].get_state(), 'child')
+        self.assertEqual(self.nodes[SED1].get_state(), 'child')
 
-        self.nodes[ED3].start()
+        self.nodes[MED3].start()
         time.sleep(5)
-        self.assertEqual(self.nodes[ED3].get_state(), 'child')
+        self.assertEqual(self.nodes[MED3].get_state(), 'child')
```

Comment:
what's your principle for bringing up devices? 
I remember that recently you would bring up all device once before waiting and checking the state :)